sass/styles.scss

```diff
@@ -0,0 +1,113 @@
+$font-main: 'Lato', sans-serif;
+$no: none;
+
+body {
+    background-color: white;
+    font-family: $font-main;
+    color: black;
+    font-size: 62.5%;
+    width: 1024px;
+    margin: 0 auto;
+}
+
+
+.description p {
+    font-size: 1.8em;
+    line-height: 2em;
+}
+
+header {
+    margin: 0 auto;
+    background-color: #F6F6F6;
+}
+
+.head {
+    background-color: #F6F6F6;
+    margin-bottom: 100px;
+    display: block;
+}
+
+th {
+    color: #B26D80;
+    text-decoration: $no;
+    font-size: 2.8em;
+    text-align: center
+}
+
+h1 {
+    font-size: 5.8em;
+    line-height: .6em;
+}
+
+h2 {
+    font-size: 4.0em;
+    line-height: 0em;
+}
+
+h3 {
+    font-size: 2em;
+    color: #AFAFAF;
+    line-height: 1em;
+}
+.name {
+    margin-left: 20px;
+    margin-top: -25px;
+}
+
+.mica {
+    float: left;
+    margin-top: 100px;
+    margin-right: 30px;
+}
+
+.welcome {
+    float: left;
+    margin-top: 100px;
+}
+
+.social-media {
+    float:right;
+}
+
+.logo {
+    float: left;
+}
+
+.resume a {
+    color:  #B26D80;
+    font-style: italic;
+    font-size: 1.6em;
+    line-height: .2em;
+}
+
+td p {
+    font-size: 1.8em;
+    margin-bottom: 20px;
+}
+
+td a {
+    color: #787878;
+    text-decoration: $no;
+}
+
+tr img {
+    margin-right: 24px;
+}
+
+footer {
+    height: 100px;
+    background-color: #B26D80;
+}
+
+.back {
+    font-weight: bold;
+    float: left;
+    margin-top: 100px;
+    text-decoration: $no;
+    font-size: 1.8em;
+}
+
+.back a {
+     text-decoration: $no;
+     color: #B26D80;
+}
```

Comment:
As succint as this stylesheet is I don't see any obvious examples of where mixins could be used. But one example is to create one for `float: left;` You'd do this (probably at the top of the file):

```scss
@mixin float() {
  float: left;
}
```

And then anywhere in the document you want to use this you do this: 

```scss
.logo {
  @include float;
}
```

This makes a bigger difference if you have several settings you use in combination, such as if you also set a width with float:

```scss
@mixin float($width:auto, $direction:left) {
  float: $direction;
  width: $width;
}
```
Then if you wanted `.logo` to be say `100px` width you could do this:

```scss
.logo {
  @include float(200px);
}
```

The result when compiled would be this:

```scss
.logo {
  float: left;
  width: 200px;
}
```

There should be opportunities to practice this on your next project, but consider whether you have cases for this kind of tool here too.

Reply:
Let's tuck away for Project 3 if not earlier that a technology you should get up to speed on is flexbox.